packages/core/rollup.config.js

```diff
@@ -67,8 +67,7 @@ export default {
     index: path.join(SRC_PATH, "index.js"),
     icons: path.join(SRC_PATH, "components/Icon/Icons/index.ts"),
     interactionsTests: path.join(SRC_PATH, "tests/interactions-utils.ts"),
-    testIds: path.join(SRC_PATH, "tests/test-ids-utils.ts"),
-    next: path.join(SRC_PATH, "next/next.ts")
```

Comment:
dont we also need to add this line back?

Reply:
Good catch, fixing